src/WebManager/Controllers/ApiController.cs

```diff
@@ -93,7 +122,10 @@ public RegistryCredential GetDockerCredential()
             {
                 return new UnauthorizedResult();
             }
-
+            
+            //These Headers are added because now any application thats needs these information can get it through an http request.
+            Response.Headers.Add("Docker-Content-Digest", resp.Item3);
```

Comment:
It's not clear what `resp.Item3` is. Will this `Item3` reference always hold the `"Docker-Content-Digest", or could this change in the future? (say, if the order of the elements are changed).

Please replace the use of a Tuple for a strongly-typed object.